packages/grafana-data/typings/jest/index.d.ts

```diff
@@ -6,6 +6,11 @@ declare global {
   namespace jest {
     interface Matchers<R, T = {}> {
       toEmitValues<E = ObservableType<T>>(expected: E[]): Promise<CustomMatcherResult>;
+      /**
```

Comment:
@hugohaggmark does it makes sense to add this here? It does not seem autogenerated but not sure. Why I am adding it here and also in the implementation is because any IDE navigation will bring me here, have to search for the implementation which is bit tedious.

Reply:
Looks nice!